sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -503,6 +502,7 @@ class RapidsExecutorPlugin extends ExecutorPlugin with Logging {
   }
 
   override def shutdown(): Unit = {
+    GpuTimeZoneDB.shutdown()
```

Comment:
the more of these shutdown calls we have the higher is the risk that one of them will throw and leave the remaining ones up.

Reply:
Understood. FWIW, `GpuTimeZoneDB.shutdown()` implementation is actually within a try block. So it should fail silently and pass to the other calls.

See https://github.com/NVIDIA/spark-rapids-jni/pull/1553 and `GpuTimeZoneDB.java` there.

Reply:
do you mean this?

https://github.com/NVIDIA/spark-rapids-jni/pull/1553/files#diff-08f7ad4a7c0bb6b12259f2f005bc3489e6785d29acc1ff8f2289c3557d6480e6R221-R225